src/main/java/com/prototype/booking/api/referencedata/RoomsInfoRepository.java

```diff
@@ -0,0 +1,7 @@
+package com.prototype.booking.api.referencedata;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.JpaSpecificationExecutor;
+
+public interface RoomsInfoRepository extends JpaRepository<RoomsInfo, String>, JpaSpecificationExecutor<RoomsInfo> {
```

Comment:
what is the purpose of **JpaSpecificationExecutor**, any plan to use it later? 
same applies to **TimingsRepository**

Reply:
Yes we use for to retrieve queries using spa specifications.